chart/templates/_helpers.tpl

```diff
@@ -0,0 +1,148 @@
+{{- define "deployment.name" -}}
+{{ .Values.fullnameOverride | default .Release.Name | trunc 63 | trimSuffix "-"}}
+{{- end -}}
+
+{{- define "deployment.labels" -}}
+app: {{ include "deployment.name" $ | quote }}
+chart-name: legend-deployment
+chart-version: {{ .Chart.Version | quote }}
+{{- range $k, $v := .Values.global.labels }}
+{{ printf "%s: %s" $k ($v | quote) }}
+{{- end -}}
+{{- end -}}
+
+{{- define "deployment.dataDogAnnotations" -}}
+{{- if .Values.deployment.dataDog.enableLogs }}
+ad.datadoghq.com/{{ include "deployment.name" $ }}.logs: '[{"source": "{{ include "deployment.name" $ }}", "service": "{{ include "deployment.name" $ }}"}]'
+tags.datadoghq.com/env: {{ .Values.global.environment | quote }}
```

Comment:
This can be removed now. It was used on the legacy on-prem due to prod-test/test being one cluster. Now we have 3 clusters (one per environment) and is configured on Agent level. 
Change in this PR or future, your choice :)

Reply:
This specific tag: 
`tags.datadoghq.com/env: {{ .Values.global.environment | quote }}`

Reply:
Was this changed in a branch in the original repo? Should i have copied some other branch than master?

Reply:
@michaelbui99 should be removed in newest commit.